packages/core/src/actions/execute.ts

```diff
@@ -14,122 +13,277 @@ import {
   decodeExecuteLeafData,
 } from '@sphinx-labs/contracts'
 import ora from 'ora'
+import { TransactionReceipt, ethers } from 'ethers'
 
 import {
-  MerkleRootState,
-  MerkleRootStatus,
-  HumanReadableAction,
-  HumanReadableActions,
+  CompilerConfig,
+  ConfigArtifacts,
+  ParsedConfig,
+  buildDeploymentWithCompilerConfigs,
+} from '../config'
+import {
+  ApproveDeployment,
   EstimateGas,
   ExecuteActions,
-  ApproveDeployment,
+  HumanReadableAction,
+  HumanReadableActions,
+  MerkleRootState,
+  MerkleRootStatus,
 } from './types'
+import { ExecutionMode } from '../constants'
+import { SphinxJsonRpcProvider } from '../provider'
 import {
   addSphinxWalletsToGnosisSafeOwners,
   findLeafWithProof,
-  fundAccountMaxBalance,
   getGasPriceOverrides,
   getMaxGasLimit,
   getReadableActions,
-  removeSphinxWalletsFromGnosisSafeOwners,
   setManagedServiceRelayer,
-  signMerkleRoot,
-  stringifyMerkleRootStatus,
 } from '../utils'
-import { SphinxJsonRpcProvider } from '../provider'
-import { ExecutionMode } from '../constants'
-import { convertEthersTransactionReceipt } from '../artifacts'
+import { shouldBufferExecuteActionsGasLimit } from '../networks'
 import {
   SphinxTransactionReceipt,
   ensureSphinxAndGnosisSafeDeployed,
 } from '../languages'
-import { ParsedConfig } from '../config'
-import { shouldBufferExecuteActionsGasLimit } from '../networks'
-
-export const executeDeployment = async (
```

Comment:
I completely removed this function since the website will now also use the `compileAndExecuteDeployment` function.